### DISCUSSION TITLE
Detailed info attached to Bolt cards

### DISCUSSION THREAD

**Discussion by gemelnik** [btcpayserver/btcpayserver#6175] [Ideas / Feature Requests]
It would be great if when programming BoltCards in the Card Factory, we can:

1. See a list of all cards that have been programmed
2. Add a tag, name, or number to each card (for our case, we would add the name/username or card number of the person who bought the card)

**Comment by ndeet** [comment]
Sounds like good features to have, so for 2. it would be a text field you can enter some value. 

Might be better to open an issue in the plugin repository here https://github.com/NicolasDorier/boltcards-plugin/issues

**Reply by gemelnik** [reply]
Just did so!:) -> https://github.com/NicolasDorier/boltcards-plugin/issues/15

**Comment by pavlenex** [comment]
Closing since it's tracked on plugin already.
